go/Internal/commands/deleteSongCommand.go

```diff
@@ -0,0 +1,73 @@
+package internal
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/Canhassi12/scylla-cloud-getting-started/Internal/database"
+	"github.com/scylladb/gocqlx/v2"
+)
+
+type DeleteSongCommandInterface interface {
+	Delete(*gocqlx.Session) error
+}
+
+type DeleteSongCommand struct {
+
+}
+
+func NewDeleteSongCommand() *DeleteSongCommand {
+	return &DeleteSongCommand{}
+}
+
+func (cmd *DeleteSongCommand) Delete(session *gocqlx.Session) error {
+	songs, err := NewListSongCommand().List(session); if err != nil {
```

Comment:
I think that what you want to say here is:

```go
if songs, err := NewListSongCommand().List(session); err != nil {
```

But, if you change it to have all methods inside the `SongController` it would be:

```go
if songs, err := c.List(); err != nil {
```